README.md

```diff
@@ -1,4 +1,21 @@
 # Getting Started
+Our dev environment can be used in two ways
+### Docker:
```

Comment:
**nitpick:** Would prefer semantically correct subheadings

A 1st level heading should be followed by a 2nd level subheading.